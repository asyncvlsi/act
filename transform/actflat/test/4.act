defcell blah(bool? a; bool! b)
{
  bool _b;
  prs {
    a => _b-
   _b => b-
  }
}

defcell buf <: blah() { }


defproc foo(bool? x_0, x_1, x_2)
{
 buf u[3];
 x_0 = x_1;
 u[0].a = x_0;
 x_1 = x_2;
}
