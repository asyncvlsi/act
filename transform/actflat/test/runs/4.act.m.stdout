defcell buf (bool a;bool b)
{
 a=a;
 b=b;

/* instances */

/* connections */
prs {
a => _b-
_b => b-
}
}


defproc foo (bool x__0)
{
 /* nets for  -toplevel-*/
 bool u_50_6_ab;
 bool u_51_6_aa;
 bool u_51_6_ab;
 bool u_52_6_aa;
 bool u_52_6_ab;
 buf u_50_6 (x__0,u_50_6_ab);
 buf u_51_6 (u_51_6_aa,u_51_6_ab);
 buf u_52_6 (u_52_6_aa,u_52_6_ab);
}
